src/openssl/lib_crypto.cr

```diff
@@ -1,5 +1,8 @@
 @[Link(ldflags: "`command -v pkg-config > /dev/null && pkg-config --libs libcrypto || printf %s '-lcrypto'`")]
 lib LibCrypto
+  OPENSSL_102 = {% `command -v pkg-config > /dev/null && pkg-config --atleast-version=1.0.2 libcrypto && printf %s true` == "true" %}
+  OPENSSL_110 = {% `command -v pkg-config > /dev/null && pkg-config --atleast-version=1.1.0 libcrypto && printf %s true` == "true" %}
```

Comment:
Figured out why CI fails, the assignment this way is runtime and thus always truthy at compile time, since it expands to the AST node. Doing an assignment to a constant at compile time seems  to be not (no longer?) possible :(